mappings/net/minecraft/nbt/AbstractNbtNumber.mapping

```diff
@@ -0,0 +1,32 @@
+CLASS net/minecraft/class_2514 net/minecraft/nbt/AbstractNbtNumber
+	COMMENT Represents an NBT number.
+	COMMENT <p>
+	COMMENT The type value {@code 99} is an alias for elements representing numbers.
```

Comment:
I am a bit confused about this line. Is this just the NBT type returned by `AbstractNbtNumber`, or does it have some other functionality? Please clarify this.

Reply:
it's used as a parameter for getting subtags i think so you can get a number from tag of any number type

Reply:
It's only used for checking the existence of an element of the given type in `NbtCompound`